fiftyone/factory/repos/execution_store.py

```diff
@@ -1,35 +1,30 @@
 """
 Execution store repository.
+
+| Copyright 2017-2024, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
 """
 
-import datetime
-import bson
-from pymongo.collection import Collection
-from fiftyone.operators.store.models import StoreDocument, KeyDocument
+from datetime import datetime
 
+from bson import ObjectId
+from pymongo.collection import Collection
 
-def _where(store_name, key=None, dataset_id=None):
-    query = {"store_name": store_name}
-    if key is not None:
-        query["key"] = key
-    if dataset_id is not None:
-        query["dataset_id"] = dataset_id
-    return query
+from fiftyone.operators.store.models import StoreDocument, KeyDocument
 
 
-class ExecutionStoreRepo:
+class ExecutionStoreRepo(object):
     """Base class for execution store repositories."""
 
     COLLECTION_NAME = "execution_store"
 
-    def __init__(
-        self, collection: Collection, dataset_id: bson.ObjectId = None
-    ):
+    def __init__(self, collection: Collection, dataset_id: ObjectId = None):
         self._collection = collection
         self._dataset_id = dataset_id
 
     def create_store(self, store_name, permissions=None) -> StoreDocument:
```

Comment:
While we're at it - should remove reference to permissions

Reply:
Done in https://github.com/voxel51/fiftyone/pull/5086/commits/0ffb9838b0b8f457a1ddd9e16cb341053c4a29ec